### ISSUE TITLE
Reference column name is blank when creating one to one link resulting in `400` error response

### ISSUE THREAD

**Issue by silentninja** [centerofci/mathesar#1767]
## Description
When creating a one-to-one link using the UI, the server responds with 

```
[
    {
        "code": 2031,
        "field": "reference_column_name",
        "message": "This field may not be blank.",
        "detail": {}
    }
]
```
This error occurs even though the frontend has a field to enter the `reference column name` on UI. The request sent to the backend is 
```
{
    "link_type": "one-to-one",
    "reference_table": 172,
    "reference_column_name": "",
    "referent_table": 44
}
```

## Expected behavior
The `reference column name` entered by the user should be sent to the backend.

## To Reproduce
- Create a one-to-one link using the `Link Table` action in the Table Action Pane.

## Additional context

UI form along with the data.

![staging mathesar org_mathesar_tables_12_tables_44_](https://user-images.githubusercontent.com/4469754/194669251-f4d4eb38-3941-4742-8266-b058c13eea00.png)

**Comment by kgodey** [comment]
Probably a duplicate of https://github.com/centerofci/mathesar/issues/1685

**Comment by silentninja** [comment]
Closed it as it is a duplicate of #1685
